Example/Backpack/SceneDelegate.swift

```diff
@@ -34,6 +34,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window.rootViewController = rootViewController(isUITestRun: isUITestRun)
         if ThemeHelpers.forceDarkMode() {
             window.overrideUserInterfaceStyle = .dark
+        } else {
+            window.overrideUserInterfaceStyle = .light
```

Comment:
I think this shouldn't be committed

Reply:
@frugoman Ah, that's a good question. 
I can remove it if you think it shouldn't be here. However, this line is helping with stability for snapshots. As we're setting only dark mode when there is a dark mode parameter in arguments. However, it can be a dark mode already on the device by default sometime, which makes screenshot for a light mode captured in a dark mode.
So, this line is just for better reliability here. I even noticed there was a wrong screenshot in the repo which was fixed by this change.
But if there is other complications which I don't see here, just let me know please!
<img width="1114" alt="image" src="https://github.com/Skyscanner/backpack-ios/assets/7831380/ab207912-d497-443f-9c9d-bb89c1894387">

Reply:
snapshots are now taken on CI actually! so not really a problem anymore 😁

Reply:
ah you mean the screenshots, yeah then this is fine!